source/includes/fact-minimum-server-version-support.rst

```diff
@@ -1 +1 @@
-The minimum supported server versions of MongoDB are 6.0.13 and 7.0.8.
+The minimum supported server versions of MongoDB are 6.0.16 and 7.0.9.
```

Comment:
**Question**

I wonder if we should add [a minimum-supported and maximum supported variables](https://github.com/10gen/docs-mongodb-internal/blob/master/snooty.toml#L289-L290) in `snooty.toml` similar to how we do with `latest-lts-version` and `last-supported-version`. WDYT? 

An advantage to doing this would be a consistent way to manage these type of version based declarations in all doc sets.

[Example syntax](https://github.com/10gen/docs-mongodb-internal/blob/master/source/reference/configuration-options.txt#L23)

Reply:
That could be helpful! I think my main concern, though is the fact that different c2c versions have different supported versions that are continuously updating, so it might end up being the same amount of work.

Reply:
Sure if thats the case I dont know C2C as well as you so will defer for this! Just wanted to throw it out there.